bundleserver/src/main/java/net/discdd/server/bundlerouting/BundleRouting.java

```diff
@@ -69,10 +69,11 @@ public void processClientMetaData(Path payloadPath, String transportID, String c
             throw new RoutingExceptions.ClientMetaDataFileException("Error Reading client metadata: " + e);
         }
 
-        if (clientMap == null || clientMap.keySet().isEmpty()) {
-            logger.log(WARNING, "[BundleRouting]: Client Metadata is empty, initializing transport with score 0");
-            updateEntry(transportID, clientID, 0);
-        } else {
+        logger.log(WARNING, "[BundleRouting]: Client Metadata is empty, initializing transport with score 0");
```

Comment:
this is not correct since we haven't checked the metadata. we probably just want to remove this line.